pkg/apis/policy/types.go

```diff
@@ -304,6 +308,12 @@ type AllowedFlexVolume struct {
 	Driver string
 }
 
+// AllowedCSIDriver represents a single CSI Driver that is allowed to be used.
+type AllowedCSIDriver struct {
+	// Name of the CSI driver
+	Name string
```

Comment:
not seeing any validation of this... Name should be required, and ensured to conform to well-formed CSI driver names (valid length and characters)